src/Microsoft.Health.Fhir.Api/Features/Audit/AuditHelper.cs

```diff
@@ -84,5 +92,30 @@ private void Log(AuditAction auditAction, HttpStatusCode? statusCode, HttpContex
                     customHeaders: _auditHeaderReader.Read(httpContext));
             }
         }
+
+        /// <summary>
+        /// Return all the values of constants of the specified type
+        /// </summary>
+        /// <param name="type">Type to examine</param>
+        /// <returns>List of constant values</returns>
+        public static IList<string> GetConstants(System.Type type)
```

Comment:
What is the thought behind having type as an argument? This method is hardcoded to only work for FhirAnonymousOperationType, it doesn't look like it should be used for any other type.

Reply:
Updated as per suggestions